### DISCUSSION TITLE
ModCon Contest Submissions

### DISCUSSION THREAD

**Discussion by jackos** [modular/modular#1293] [General]
# ModCon Contest Submissions

Submissions have closed!

You can see all the [finalists and other submissions here](https://github.com/modularml/mojo/blob/main/examples/blogs-videos/modverse-weekly/issue-11.md)

**Comment by helehex** [comment]
My submission is `Moplex ⏣`, generalized complex numbers for mojo.
https://github.com/helehex/moplex
